classes/Translations.php

```diff
@@ -0,0 +1,156 @@
+                    'allow_url_fopen' => $this->trans('PHP\'s "allow_url_fopen" option is turned on, or cURL is installed', 'Preupgrade'),
+                ],
+                'modules' => [
+                    'title' => $this->trans('Module compatibility:', 'Preupgrade'),
+                    'description' => $this->trans('As non-native modules can experience some compatibility issues, we recommend to disabled them by default. Keepin them enable might prevent you from loading the "Modules" page properly after th upgrade. Check also the %slist of native modules in 1.7%s to make sure lorem loris.', 'Preupgrade'),
+                    'help' => $this->trans('Modules will be placed in a folder for lorem ipsum atmerit', 'Preupgrade'),
+                    'list' => [
+                        'compatibility' => $this->trans('I understand that not all my modules might be compatible with the version I\'m going to upgrade to', 'Preupgrade'),
+                        'native_modules' => $this->trans('I understand that some of my native modules might loose previous data after upgrade.', 'Preupgrade'),
+                        'experience' => $this->trans('I understand  that I might experience some lorem problems blanditis vuluptatum', 'Preupgrade'),
+                    ]
+                ],
+                'core' => [
+                    'title' => $this->trans('Modified core files:', 'Preupgrade'),
+                    'description' => $this->trans('The following core files have been modified, modifications will be lost during the upgrade.', 'Preupgrade'),
+                    'list' => [
+                        'understand' => $this->trans('I understand that blanditis praesentium voluptatum files', 'Preupgrade'),
+                    ]
+                ],
+                'modal' => [
+                    'start' => [
+                        'title' => $this->trans('Start your upgrade now?', 'Preupgrade'),
+                        'description' => $this->trans('Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium dolorempque', 'Preupgrade'),
+                    ]
+                ],
+                'buttons' => [
+                    'backup' => $this->trans('One click back-up', 'Preupgrade'),
+                    'disableModules' => $this->trans('Disable all modules', 'Preupgrade'),
+                    'upgrade' => $this->trans('Upgrade PrestaShop', 'Preupgrade'),
+                    'maintenance' => $this->trans('Switch to maintenance mode', 'Preupgrade'),
+                ],
+            ],
+            'upgrade' => [
+                'title' => $this->trans('Upgrade processing', 'Upgrade'),
+                'description' => $this->trans('Depending on your lorem this can take up to lorem minutes', 'Upgrade'),
+            ],
+        ];
+
+        return $translations;
+    }
+
+    protected function trans($message, $domain, $params = [])
```

Comment:
replace `$domain` with `$moduleDomain`, `$domainEnd` or `$domainSuffix` to avoid confusion with the real full domain